Tactics/Attr.lean

```diff
@@ -8,3 +8,8 @@ initialize
 
   -- enable tracing for `sym_n` tactic and related components
   registerTraceClass `Tactic.sym
+
+  registerOption `Tactic.sym.debug {
+    defValue := true
+    descr := "enable/disable extra validation of `sym_n` internal state"
```

Comment:
"validation" is an interesting but perhaps overloaded word that could also mean "proof". Would "inspection" be more suitable, or am I missing something?

Reply:
This option does not control tracing, it causes `AxEffects.validate` to be called at certain points, which will typecheck the various expressions stored in it, throwing an error if some expression is in fact mal-formed.

That seems to me to better fit "validatiaon" than "inspection" (the latter would imply tracing to me)

Reply:
Indeed. I haven't reached `AxEffects` yet, so found the comment confusing. It'd help to add what you said to the description up above.

Reply:
I've updated the description, it now reads:
```
  -- enable extra checks for debuggins `sym_n`,
  -- see `AxEffects.validate` for more detail on what is being type-checked
  registerOption `Tactic.sym.debug {
    defValue := true
    descr := "enable/disable type-checking of internal state during execution \
      of the `sym_n` tactic, throwing an error if mal-formed expressions were \
      created"
  }

```